### ISSUE TITLE
Issue with V1.0 -  'test_selected' is undefined - incorrectly referenced in create_constraints.sql

### ISSUE THREAD

**Issue by RonniePitts** [Snowflake-Labs/dbt_constraints#72]
The latest package (v1.0) is now throwing test_selected when running the '{{dbt_constraints.create_constraints()}}' on dbt v1.6 (cloud)

**Comment by Tonayya** [comment]
Hi @sfc-gh-dflippo, I think [this PR](https://github.com/Snowflake-Labs/dbt_constraints/commit/94cefff86fd3b21774bbd83d548dda3dc1891f33) might be the issue. The error says:
``` 'test_selected' is undefined```
How and where does this need to be defined?

**Comment by RonniePitts** [comment]
Macros have been incorrectly referenced in create_constraints.sql, namely the following : 

```
{%- macro test_selected(test_model) -%}
{%- endmacro -%}

{%- macro lookup_should_rely(test_model) -%}
{%- endmacro -%}

{%- macro should_always_create_constraint(test_model) -%}
{%- endmacro -%}
```

For example 

` {%- set selected = test_selected(test_model) -%}
`

should be 

` {%- set selected = dbt_constraints.test_selected(test_model) -%}
`

**Comment by RonniePitts** [comment]
**Temporary Solution**

Create Local .sql macro with the following : 


{#- This macro checks if a test or its model is selected -#}
{%- macro test_selected(test_model) -%}
   {% do dbt_constraints.test_selected(test_model) %}
{%- endmacro -%}

{%- macro lookup_should_rely(test_model) -%}
   {% do dbt_constraints.lookup_should_rely(test_model) %}
{%- endmacro -%}


{%- macro should_always_create_constraint(test_model) -%}
   {% do dbt_constraints.should_always_create_constraint(test_model) %}
{%- endmacro -%}

{%- macro lists_intersect(listA, listB) -%}
   {% do dbt_constraints.lists_intersect(listA, listB) %}
{%- endmacro -%}

**Comment by sfc-gh-dflippo** [comment]
THANKS TO YOU BOTH for quickly reporting the issue. I reviewed @RonniePitts 's changes and made the same in the main branch.

I have released version 1.0.1 to fix the issue. I did my own tests but can you please confirm your issues are resolved with v1.0.1?

Thanks again!

**Comment by RonniePitts** [comment]
@sfc-gh-dflippo Dan, it's fixed now with the latest version.
